java/install_jdk

```diff
@@ -40,18 +41,22 @@ rm -rf ${JAVA_HOME}/*src.zip \
        ${JAVA_HOME}/jre/plugin \
        ${JAVA_HOME}/jre/lib/deploy* \
        ${JAVA_HOME}/jre/lib/plugin.jar \
-       ${JAVA_HOME}/jre/lib/*javafx* \
-       ${JAVA_HOME}/jre/lib/*jfx* \
        ${JAVA_HOME}/jre/lib/amd64/libdecora_sse.so \
-       ${JAVA_HOME}/jre/lib/amd64/libprism_*.so \
        ${JAVA_HOME}/jre/lib/amd64/libfxplugins.so \
        ${JAVA_HOME}/jre/lib/amd64/libglass.so \
        ${JAVA_HOME}/jre/lib/amd64/libgstreamer-lite.so \
-       ${JAVA_HOME}/jre/lib/amd64/libjavafx*.so \
-       ${JAVA_HOME}/jre/lib/amd64/libjfx*.so \
-       ${JAVA_HOME}/jre/lib/ext/jfxrt.jar \
        ${JAVA_HOME}/jre/lib/ext/nashorn.jar \
        ${JAVA_HOME}/jre/lib/oblique-fonts
 
+if [ -n "${REMOVE_JFX}" ] ; then
+  echo "Removing JavaFX framework and libraries..."
+  rm -rf ${JAVA_HOME}/jre/lib/amd64/libprism_*.so \
+       ${JAVA_HOME}/jre/lib/*javafx* \
+       ${JAVA_HOME}/jre/lib/*jfx* \
+       ${JAVA_HOME}/jre/lib/amd64/libjavafx*.so \
+       ${JAVA_HOME}/jre/lib/amd64/libjfx*.so \
+       ${JAVA_HOME}/jre/lib/ext/jfxrt.jar
```

Comment:
Sorry, I misread my own diff. These are actually all the ones needed for JavaFx:
`        ${JAVA_HOME}/jre/lib/amd64/libdecora_sse.so \
         ${JAVA_HOME}/jre/lib/amd64/libfxplugins.so \
         ${JAVA_HOME}/jre/lib/amd64/libglass.so \
         ${JAVA_HOME}/jre/lib/amd64/libgstreamer-lite.so \
        ${JAVA_HOME}/jre/lib/amd64/libprism_*.so \
        ${JAVA_HOME}/lib/*javafx* \
        ${JAVA_HOME}/jre/lib/*javafx* \
        ${JAVA_HOME}/jre/lib/*jfx* \
         ${JAVA_HOME}/jre/lib/amd64/libjavafx*.so \
         ${JAVA_HOME}/jre/lib/amd64/libjfx*.so \
        ${JAVA_HOME}/jre/lib/ext/jfxrt.jar`